.github/workflows/ci.yml

```
@@ -19,44 +16,43 @@ jobs:
           docker pull ubuntu:18.04
           docker pull "dependabot/dependabot-core:latest"
           docker pull "dependabot/dependabot-core-ci:latest" || true
-      - name: Build dependabot-core image
-        run: |
-          docker build \
-            -t "dependabot/dependabot-core:latest" \
-            -t "dependabot/dependabot-core:${{ env.VERSION }}" \
-            --cache-from ubuntu:18.04 \
-            --cache-from "dependabot/dependabot-core:latest" \
-            .
       - name: Build dependabot-core-ci image
         run: |
           rm .dockerignore
           docker build \
             -t "dependabot/dependabot-core-ci:${{ github.sha }}" \
-            -t "dependabot/dependabot-core-ci:latest" \
             -f Dockerfile.ci \
             --cache-from ubuntu:18.04 \
             --cache-from "dependabot/dependabot-core:latest" \
             --cache-from "dependabot/dependabot-core-ci:latest" \
             .
-      - name: Push dependabot-core-ci image to CI cache repo
-        run: |
-          docker push "dependabot/dependabot-core-ci:latest"
-          docker push "dependabot/dependabot-core-ci:${{ github.sha }}"
+          docker image save -o dependabot-core-ci-image.tar "dependabot/dependabot-core-ci:${{ github.sha }}"
+      - name: Save image as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: dependabot-core-ci-image.tar
+          path: dependabot-core-ci-image.tar
 
   rubocop:
     name: Rubocop
     runs-on: ubuntu-latest
-    container: dependabot/dependabot-core-ci:${{ github.sha }}
     needs: ci-image
     steps:
+      - name: Load docker image from artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: dependabot-core-ci-image.tar
+      - name: Restore image
+        run: |
+          docker image import dependabot-core-ci-image.tar "dependabot/dependabot-core-ci:${{ github.sha }}"
```

Perhaps this helps?


```suggestion
          docker load < dependabot-core-ci-image.tar
```

For reference, see https://pspdfkit.com/blog/2019/docker-import-export-vs-load-save/.